mod/wet4/views/default/page/elements/head.php

```diff
@@ -66,13 +66,14 @@
     }
 
   }
+
 } else {
 
   $current_url = explode('/', $_SERVER['REQUEST_URI']);
   $page_entity = get_entity($current_url[count($current_url) - 1]);
 
   if ($elgg_entity instanceof ElggUser || $elgg_entity instanceof ElggGroup) {
```

Comment:
Aside: Just realized this conditional should be split since you're doing a ternary using the conditions inside.
```php
if ($elgg_entity instanceof ElggUser) {
  $page_title = $vars['title'];
} elseif ($elgg_entity instanceof ElggGroup) {
  $page_title = gc_explode_translation($elgg_entity->name, $gc_language) . " : {$vars['title']}";
} else {
...
```